noir-projects/noir-contracts/contracts/schnorr_single_key_account_contract/src/util.nr

```diff
@@ -3,18 +3,19 @@ use dep::aztec::protocol_types::address::PublicKeysHash;
 use dep::std::{schnorr::verify_signature_slice};
 use crate::auth_oracle::AuthWitness;
 
-pub fn recover_address(message_hash: Field, witness: AuthWitness) -> AztecAddress {
-    let message_bytes = message_hash.to_be_bytes(32);
-    let verification = verify_signature_slice(
-        witness.owner.x,
-        witness.owner.y,
-        witness.signature,
-        message_bytes
-    );
-    assert(verification == true);
+// TODO(#5830): the following is currently broken because we are no longer able to compute public keys hash
```

Comment:
Commented this out now since I refactored PublicKeysHash and I plan on tackling this func in #5830